docs/security/users-and-teams/auditing.md

```diff
@@ -6,6 +6,10 @@ position: 30
 
 For team members to collaborate in the deployment of software, there needs to be trust and accountability. Octopus Deploy captures audit information whenever significant events happen in the system.
 
+:::hint
+The [Audit Retention functionality](#manage-archived-audit-events) was introduced in **Octopus 2022.3** and is being rolled out in stages for our Cloud customers first. It will be made available to Self-hosted customers at a later stage.
```

Comment:
```suggestion
The [Audit Retention functionality](#manage-archived-audit-events) was introduced in **Octopus 2022.4** and is being rolled out in stages for our Cloud customers first. It will be made available to Self-hosted customers at a later stage.
```

Reply:
According to the latest information I saw, this would now be introduced in 2022.4